util/settings/settings.go

```diff
@@ -816,7 +816,7 @@ func (a *ArgoCDSettings) GetAppInstanceLabelKey() string {
 
 func (a *ArgoCDSettings) getExcludedResources() []ExcludedResource {
 	coreExcludedResources := []ExcludedResource{
-		{ApiGroups: []string{"events.k8s.io", "metrics.k8s.io"}, Kinds: []string{"*"}, Clusters: []string{"*"}},
+		{ApiGroups: []string{"events.k8s.io", "metrics.k8s.io"}},
```

Comment:
Just noticed this, golang style guide suggests this field name be named: APIGroups.

Reply:
Good spot. I'll fix this too.